### ISSUE TITLE
Rename account button needs better placement.

### ISSUE THREAD

**Issue by jrick** [decred/Paymetheus#64]
At the moment, this is the only button that can be used to interact with any of the listed accounts.  It is currently placed below the account properties and looks very silly all alone.  As long as there is only one button, it needs to be moved somewhere better.

**Comment by marcopeereboom** [comment]
Yes please.  At the very least it needs to be renamed from Aa to "rename account" for 0.1.0.
